lib/gemojione.rb

```diff
@@ -75,7 +75,7 @@ def self.image_url_for_unicode_moji(moji)
   def self.image_tag_for_moji(moji)
     if use_sprite
       emoji = index.find_by_moji(moji)
-      %Q{<span class="emojione-#{emoji['unicode'].to_s.downcase}" alt="#{moji}" title="#{moji}">⃣#{moji}</span>}
+      %Q{<span class="emojione emojione-#{emoji['unicode'].to_s.downcase}" alt="#{ moji }" title="#{ moji }">#{ moji }</span>}
```

Comment:
Removing the `⃣` from `...title="#{moji}">⃣#{moji}</span>` should be part of the previous commit. That square is the problem in some failing tests, it shouldn't be there nor in the tests.